### ISSUE TITLE
Hitting ESC within the Link Control in the list view seems to completely loose focus.

### ISSUE THREAD

**Issue by getdave** [WordPress/gutenberg#48015]
Add Custom Link then once Link UI shows up hit ESC. Notice focus is now on the `body` or `iframe`. Should be back where it came from.

**Comment by scruffian** [comment]
I think this is also fixed by https://github.com/WordPress/gutenberg/pull/48029.

**Comment by draganescu** [comment]
I have tested this and when closing the popover from the linkUI `document.activeElement ` is set to `<button class="components-button block-inserter...` which is the element of the square inserter at the bottom.
